backend-lib/src/main/rust/lib.rs

```diff
@@ -552,18 +552,15 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_importAcc
                 }
             })?;
 
-        let account_name = java_string_to_rust(env, &account_name);
-        let key_source = java_nullable_string_to_rust(env, &key_source);
+        let account_name = java_string_to_rust(env, &account_name)?;
+        let key_source = java_nullable_string_to_rust(env, &key_source)?;
 
         let purpose = match purpose {
             0 => {
-                let seed_fingerprint = if !seed_fingerprint_bytes.is_null() {
-                    <[u8; 32]>::try_from(&env.convert_byte_array(seed_fingerprint_bytes)?[..])
-                        .ok()
-                        .map(SeedFingerprint::from_bytes)
-                } else {
-                    None
-                };
+                let seed_fingerprint =
+                    utils::java_nullable_bytes_to_rust(env, &seed_fingerprint_bytes)?
+                        .and_then(|b| b.as_slice().try_into().ok())
+                        .map(SeedFingerprint::from_bytes);
```

Comment:
Although this happens to only be used once, it's doing the same kind of thing as the other conversion methods, so I might be inclined to add a helper anyway.